esdt-safe/tests/bridge_blackbox_tests.rs

```diff
@@ -278,6 +270,7 @@ impl BridgeTestState {
             .to(BRIDGE_ADDRESS)
             .typed(esdt_safe_proxy::EsdtSafeProxy)
             .execute_operations(hash_of_hashes, operation)
+            .returns(ExpectError(4, "Operation is not registered"))
```

Comment:
Why is this returning error? 
You have this function that returns error `propose_execute_operation_and_expect_err`.

Reply:
maybe just rename the test to `propose_execute_operation_fail` to be obvious that it expects fail..

Reply:
I forgot about that function, and realized the error that was made, I fixed the test